src/helper/html.rs

```diff
@@ -186,176 +180,98 @@ pub struct HfdElement {
     classes: Vec<String>,
 }
 impl HfdElement {
-    pub fn get_by_name(&self, name: &str) -> bool {
-        let mut result = false;
-        if self.name == name {
-            result = true;
-        }
-        result
+    pub fn has_name(&self, name: &str) -> bool {
```

Comment:
As this function only returns a bool therefore name appropriately and is a **breaking** change.